ci-operator/step-registry/cucushift/installer/check/vsphere/usertags/cucushift-installer-check-vsphere-usertags-commands.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+source "${SHARED_DIR}/govc.sh"
+source "${SHARED_DIR}/vsphere_context.sh"
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+declare vsphere_datacenter
+INFRA_ID=$(jq -r .infraID ${SHARED_DIR}/metadata.json)
+check_result=0
+TAG_FILE="${SHARED_DIR}/tags_lists"
```

Comment:
In provsion/deprovision step, it is using "USER_TAG" ENV to get the list, why not use the same ENV var in this step?